indra/config.py

```diff
@@ -136,3 +134,179 @@ def has_config(key):
     """
     return get_config(key, failure_ok=True) is not None
 
+
+KNOWLEDGE_SOURCE_INFO = {
```

Comment:
Can we move this info to a separate resource file?

Reply:
I suppose we could add a file such as `source_info.json`. I would note this is also where we could add descriptions for each of the sources and the database API is already set up to incorporate them once we add them.

Reply:
Yes, source_info.json in resources would work, and I'd actually suggest putting the KNOWLEDGE_SOURCE_INFO variable into `indra/sources/__init__.py` since `config` is meant more for user-specific environmental configurations.

Reply:
:+1: